xml/MAIN-SBP-Migrate-z-KVM.xml

```diff
@@ -130,7 +130,7 @@
                 the command line utilities to manage virtual machines. These utilities are installed
                 as part of the KVM virtualization host and tools pattern. The section
                     <quote>Virtualization Console Tools</quote> at <link
-                    xlink:href="https://www.suse.com/documentation/sles-12/book_virt/data/sec_tools_intro_console.html"
+                    xlink:href="https://documentation.suse.com/sles/12-SP4/html/SLES-all/cha-tools-intro.html#sec-tools-intro-console"
```

Comment:
...and here...

Reply:
See above regarding MAIN-SBP-Migrate-z-KVM.xml